docker-compose.yml

```diff
@@ -9,21 +9,32 @@ services:
       - weblate-data:/app/data
     env_file:
       - ./environment
+    ports:
+      - "8000:8000"
     restart: always
     depends_on:
       - database
       - cache
   web:
-    build: nginx
-    links:
-      - weblate
+    image: jwilder/nginx-proxy
+    volumes:
+      - /var/run/docker.sock:/tmp/docker.sock:ro
+      - ./certs:/etc/nginx/certs:ro
+      - /etc/nginx/vhost.d
+      - /usr/share/nginx/html
     ports:
       - "80:80"
-    volumes_from:
-      - weblate:ro
```

Comment:
Sorry, but I'm not going to merge things which make the performance worse. We're indeed now using django built in server (which can serve static files) instead of uwsgi, but that should be fixed and then this would make the situation just worse.

Overall I'm not really sure that every Docker container should have letsencrypt support, I think having one frontend container (eg. https://hub.docker.com/r/steveltn/https-portal/) handling this is better way...